server/model/game_logic_manager.cpp

```diff
@@ -1,6 +1,6 @@
 #include "game_logic_manager.h"
+#include <boost/process.hpp>
```

Comment:
мне понадобился ещё костыль

```suggestion
// workaround for a boost bug...
// https://github.com/boostorg/process/issues/96
#ifndef __kernel_entry
#define __kernel_entry
#endif
#include <boost/process.hpp>
```

Reply:
Это в итоге так и не починили?

Reply:
у меня boost 1.77, без него не работает на моей винде